nevergrad/benchmark/__main__.py

```diff
@@ -35,6 +35,13 @@ def launch(experiment: str, num_workers: int = 1, seed: Optional[int] = None,
         core.save_or_append_to_csv(df, csvpath)
     else:
         print(f"Saved data to {csvpath}")
+        print("Simple plots: python -m nevergrad.benchmark.plotting <csv_file>")
+        print("Users might want to explore their experimental data with IOH Profiler:")
+        print("1. Go to http://iohprofiler.liacs.nl/")
```

Comment:
I would rather the instructions were written in only one location (in a section of `docs/benchmarking.md`, see below), this way if anything change there will be only one place were we need to change it. You can just refer to the section directly as you did in the last print (with updated path)